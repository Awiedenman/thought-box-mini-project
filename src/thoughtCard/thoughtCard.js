import React from 'react';

export const ThoughtCard = ( { id, title, body }) => {
  return (
    <div className='thought' id={ id }>
      <h2>{ title }</h2>
      <p>{ body }</p>
      <button>Delete</button>
    </div>
  )
}
