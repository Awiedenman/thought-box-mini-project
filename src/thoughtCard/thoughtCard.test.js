import React from 'react';
import { shallow } from 'enzyme';

import { ThoughtCard } from './thoughtCard.js';

describe('ThoughtCard', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(
      <ThoughtCard
        title='title'
        body='body'
        id={1} 
      />) 

    expect(wrapper).toMatchSnapshot()
  });

});
