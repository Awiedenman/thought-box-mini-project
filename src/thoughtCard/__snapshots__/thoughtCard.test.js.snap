// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThoughtCard should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    body="body"
    id={1}
    title="title"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          title
</h2>,
        <p>
          body
</p>,
        <button>
          Delete
</button>,
      ],
      "id": 1,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "title",
        },
        "ref": null,
        "rendered": "title",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "body",
        },
        "ref": null,
        "rendered": "body",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Delete",
        },
        "ref": null,
        "rendered": "Delete",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            title
</h2>,
          <p>
            body
</p>,
          <button>
            Delete
</button>,
        ],
        "id": 1,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "title",
          },
          "ref": null,
          "rendered": "title",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "body",
          },
          "ref": null,
          "rendered": "body",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Delete",
          },
          "ref": null,
          "rendered": "Delete",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
